services/app/assets/js/widgets/slices/UIState.js

```diff
@@ -0,0 +1,32 @@
+import { createSlice } from '@reduxjs/toolkit';
+import EditorModes from '../config/editorModes';
+import EditorThemes from '../config/editorThemes';
+import TaskDescriptionLanguages from '../config/taskDescriptionLanguages';
+
+const initialState = {
+  editorMode: EditorModes.default,
+  editorTheme: EditorThemes.dark,
+  taskDescriptionLanguage: TaskDescriptionLanguages.default,
+};
+
+const UIState = createSlice({
```

Comment:
Лучше строго по конвеции и с большой буквы начинать имена классов или jsx компонентов.
```suggestion
const gameUI = createSlice({
```